There should not be promotion of abuse, even if not clearly in an "offensive comment"

nitech-code-of-conduct-suggestions-2 opened the pull request:
"Offensive comments" is too vague to call out promoting discrimination (like links or tweets to pub/controversy pieces by racist, sexist or homophic groups), that hide under "free speech". This backs us up in rejecting that kind of content when admins see it's necessary.

Could be in different place in list, but then would need characteristics list copied.

nitech-code-of-conduct-suggestions-2 committed:
Promotion of abuse and discrimination

"Offensive comments" is too vague to call out promoting discrimination (like links or tweets to pub/controversy pieces by racist, sexist or homophic groups), that hide under "free speech". This backs us up in rejecting that kind of content when admins see it's necessary.

Could be in different place in list, but then needs characteristics list copied.

mauricerkelly commented:
I think this would be better served if it included an example of what is considered to be promotion of abuse (which is actually done in this PR: "like links or tweets to pub/controversy pieces by racist, sexist or homophic groups")

nitech-code-of-conduct-suggestions-2 committed:
Give examples of what promoting abuse means

e.g. links to "two-side" stories presenting racist, sexist or other groups in an equal or positive light

mauricerkelly reviewed (changes requested):
Another couple of comments here:
- what are "pub/controversy pieces" - specifically "pub"? Maybe "controversial pieces" would be sufficient?
- is this a typo in "homophic"?

MarkXA reviewed (commented):
This should probably be moved from "Harassment" to an "Unacceptable behaviour" section. Possibly rephrase to something like "Supporting or promoting behaviour or statements that do not comply with this Code of Conduct".

nitech-code-of-conduct-suggestions-2 committed:
Correct language (typo and unhelpful shortening to "pub")

nitech-code-of-conduct-suggestions-2 commented:
Just made those changes - thank you mo for pointing out typo.

mauricerkelly committed:
Merge branch 'master' into promotion-patch

mauricerkelly commented:
I've resolved a merge conflict here. Can we get some (dis/)approvals to move it along?
